both/methods.js

```diff
@@ -185,12 +185,13 @@ Meteor.methods({
     check(competitionId, String);
     throwIfCannotManageCompetition(this.userId, competitionId);
 
-    var round = Rounds.findOne(roundId);
+    var round = Rounds.findOne(roundId); // Will not exist for some events
```

Comment:
Yeah, you're right about the exception.

I thought it wouldn't matter, but now I realize we'd add a "lunch" type event if a request with a stale roundId came in.